### PR TITLE
First

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
@EPashkin: for next releases, I intend to use this script to make it faster and safer. It's not tested yet, I wait for our next release. Don't hesitate to look around the code and check if I forgot something (like I said, I didn't test it yet).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
...
+
+ * Merge `master` branch into `crate` branch in every repository (starting with `sys` and ending with `gtk`). (/!\ `cairo-sys` is in `cairo` repository, not `sys` /!\)
+ * Update crate version in `Cargo.toml` and the number of its `gtk-rs` dependencies.
+ * Open pull requests to `crate` branches to corresponding repositories.
+ * Once pull requests have been merged, publish all crates to `crates.io` (using the `cargo publish` command).
+ * Generate new docs (don't forget `--all-features` when using `cargo doc`!). To do so, go to `gtk` directory (in `crate` branch) and run `cargo doc --features=embed-lgpl-docs`. Then run `cp -r target/doc/* ../docs` (where `docs` is the corresponding directory for the `gtk-rs/docs` repository). Commit then push the changes to the `docs` (make a pull request and all the usual stuff...).
+ * Merge `pending` branch into `master` in the `gtk-rs/examples` repository (by opening a pull request of course).
+ * Update badges version number in the `_data/crates.json` in the `gtk-rs/gtk-rs.github.io` repositoryz.
+ * Write a blog post (add the file into `_posts` folder in `gtk-rs.github.io` repository) announcing the new release.
+
```
Comment (EPashkin) [inline README.md]: * Update crate version of the `master` branches on every repository.

Comment (GuillaumeGomez) [inline README.md]: I just copied this readme from gtk/release.md. I don't know yet if I'll keep it or not.

Comment (EPashkin) [inline README.md]: Seems you copied from old version (or crate)

Comment (GuillaumeGomez) [inline README.md]: Very likely. Well, I'll rewrite a better version later on.

**Comment by EPashkin** [comment]
I'm novice in python but seems it works.

**Comment by GuillaumeGomez** [comment]
Well, we'll see that in next release. ;)

**Pull request merged**